### ISSUE TITLE
Install Error: Alpine 3.8

### ISSUE THREAD

**Issue by woodwardmatt** [nginxinc/nginx-amplify-agent#144]
Hey! I thought I''d give the Alpine install a crack on version 3.8 but spotted the following issue and thought I'd report it back: 

```
      creating build
      creating build/lib.linux-x86_64-2.7
      creating build/lib.linux-x86_64-2.7/cffi
      copying cffi/cparser.py -> build/lib.linux-x86_64-2.7/cffi
      copying cffi/vengine_gen.py -> build/lib.linux-x86_64-2.7/cffi
      copying cffi/__init__.py -> build/lib.linux-x86_64-2.7/cffi
      copying cffi/backend_ctypes.py -> build/lib.linux-x86_64-2.7/cffi
      copying cffi/ffiplatform.py -> build/lib.linux-x86_64-2.7/cffi
      copying cffi/setuptools_ext.py -> build/lib.linux-x86_64-2.7/cffi
      copying cffi/verifier.py -> build/lib.linux-x86_64-2.7/cffi
      copying cffi/recompiler.py -> build/lib.linux-x86_64-2.7/cffi
      copying cffi/api.py -> build/lib.linux-x86_64-2.7/cffi
      copying cffi/lock.py -> build/lib.linux-x86_64-2.7/cffi
      copying cffi/cffi_opcode.py -> build/lib.linux-x86_64-2.7/cffi
      copying cffi/model.py -> build/lib.linux-x86_64-2.7/cffi
      copying cffi/vengine_cpy.py -> build/lib.linux-x86_64-2.7/cffi
      copying cffi/pkgconfig.py -> build/lib.linux-x86_64-2.7/cffi
      copying cffi/commontypes.py -> build/lib.linux-x86_64-2.7/cffi
      copying cffi/error.py -> build/lib.linux-x86_64-2.7/cffi
      copying cffi/_cffi_include.h -> build/lib.linux-x86_64-2.7/cffi
      copying cffi/parse_c_type.h -> build/lib.linux-x86_64-2.7/cffi
      copying cffi/_embedding.h -> build/lib.linux-x86_64-2.7/cffi
      copying cffi/_cffi_errors.h -> build/lib.linux-x86_64-2.7/cffi
      running build_ext
      building '_cffi_backend' extension
      creating build/temp.linux-x86_64-2.7
      creating build/temp.linux-x86_64-2.7/c
      gcc -fno-strict-aliasing -Os -fomit-frame-pointer -g -DNDEBUG -Os -fomit-frame-pointer -g -DTHREAD_STACK_SIZE=0x100000 -fPIC -DUSE__THREAD -DHAVE_SYNC_SYNCHRONIZE -I/usr/include/ffi -I/usr/include/libffi -I/usr/include/python2.7 -c c/_cffi_backend.c -o build/temp.linux-x86_64-2.7/c/_cffi_backend.o
      c/_cffi_backend.c:2:20: fatal error: Python.h: No such file or directory
       #include <Python.h>
                          ^
      compilation terminated.
      error: command 'gcc' failed with exit status 1

      ----------------------------------------
  Command "/usr/bin/python2.7 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-VgUnrE/cffi/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-iyBoCp/install-record.txt --single-version-externally-managed --prefix /tmp/pip-build-env-X9euKn/overlay --compile" failed with error code 1 in /tmp/pip-install-VgUnrE/cffi/

  ----------------------------------------
Command "/usr/bin/python2.7 /root/.local/lib/python2.7/site-packages/pip install --ignore-installed --no-user --prefix /tmp/pip-build-env-X9euKn/overlay --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools>=18.5 wheel "cffi>=1.8,!=1.11.3; python_implementation != 'PyPy'"" failed with error code 1 in None
```

**Comment by woodwardmatt** [comment]
This was using the other OS recommendations over at: https://github.com/nginxinc/nginx-amplify-doc/blob/master/amplify-faq.md#21-what-operating-systems-are-supported

**Comment by woodwardmatt** [comment]
**Note**: Same on Alpine 3.9 when trying to use the Alpine 3.7 install option

**Comment by woodwardmatt** [comment]
**Note**: Same on Alpine 3.9 when trying to use the "5) Other" option

**Comment by ariutk** [comment]
You don't have libffi-dev headers for build.

Commit that solves this problem https://github.com/ariutk/docker-nginx-amplify/commit/a1913dad697c344e4b9fc56bfa141f3d45300f28
Pull request https://github.com/nginxinc/docker-nginx-amplify/pull/24

**Comment by defanator** [comment]
@woodwardmatt though there are no prebuilt packages available for Alpine, latest agent version (1.8.0) is known to work with at least Alpine 3.13, 3.14, and 3.15. You can use https://github.com/nginxinc/nginx-amplify-agent/blob/master/packages/install-source.sh for getting it to work.

See also #208.

I'm going to close this one for now.
